types/node/test/worker_threads.ts

```diff
@@ -144,7 +144,7 @@ import { createContext } from "node:vm";
     bc.close();
     bc.ref();
     bc.unref();
-    bc.onmessage = (msg: unknown) => {};
+    bc.onmessage = (msg: MessageEvent) => {};
     bc.onmessageerror = (msg: unknown) => {};
```

Comment:
Once you apply the suggested changes to worker_threads, you'll see that this now fails, as undici's MessageEvent and lib.dom's MessageEvent are not interassignable (hence the need to explicitly import the interface for @types/node's BroadcastChannel).

The following will ensure the argument type is tested in a way that's compatible with and without lib.dom:

```suggestion
    bc.onmessage = (msg) => {
        msg; // $ExpectType MessageEvent<any>
    };
    bc.onmessageerror = (msg) => {
        msg; // $ExpectType MessageEvent<any>
    };
```